tutorials/tutorial-batch-processing/tutorial-batch.md

```diff
@@ -0,0 +1,98 @@
+# Batch processing
+
+![Open batch button position](./images/open_batch.png)
+
+To open window for batch processing use marked button or use *Batch processing* position in *File* menu
+
+## Create a workflow for a batch processing
+
+![Prepare plan view](./images/prepare_plan.png)
+
+1.  Select how the mask file for segmentation will be recognized. 
+
+   a. In section *Use mask from*, Select *File* tab. 
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 1 space](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=6358262)